09-redux/classworks/Volodkova_Anna/src/components/header/Header.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import Input from '../input';
+import logo from './logo.svg';
+import './header.css';
+
+function Header(props) {
+  return (
+    <div>
+      <div className="header">
+        <img src={logo} className="logo"/>
+        <Input handleOnChangeInput={props.handleOnChangeInput} handleBtnClick={props.handleBtnClick}/>
```

Comment:
see, our Header component gets `handleBtnClick` props and passes it further. we don't want to do it as developers and that's why we use Redux